[2025-07-25T20:46:11.102+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-25T20:46:11.981+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: sample_ETL.load_order_and_delivery_data manual__2025-07-25T20:28:28.526110+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-07-25T20:46:12.051+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
